### ISSUE TITLE
Is it too easy to delete attachments? 

### ISSUE THREAD

**Issue by kjappelbaum** [cheminfo/eln.epfl.ch#26]
i was testing the importation on the sample of a collaborator and by chance hit the delete icon in the ELN. 
This made me wonder if deletion should be possible (maybe also just hide attachments) and if one at least should have a confirmation for deletion.

**Comment by lpatiny** [comment]
Yes it is too easy. I will check how to add a confirmation dialog.

**Comment by kjappelbaum** [comment]
would it be something like 

```
 async deleteAttachment(entry, attachments, options)  {
    ui.confirm('Are you sure you want to delete ' + entry.value.reference)
    .then(function(res) {
        if(!res) return;
 <code from visualizer helper> 
    });
}
```


in https://github.com/cheminfo-js/visualizer-helper/blob/693e001fdb59e87587e2c154b845587a83d49ba8/rest-on-couch/Roc.js#L620
